libs/server/Resp/Bitmap/BitmapCommands.cs

```diff
@@ -523,14 +523,16 @@ private bool StringBitField<TGarnetApi>(int count, ref TGarnetApi storageApi)
                 }
 
                 //Identify sign for number
+#pragma warning disable CA1865 Use 'string.Method(char)' instead of 'string.Method(string)' for string with single char
                 byte sign = encodingArg.StartsWith("i", StringComparison.OrdinalIgnoreCase) ? (byte)BitFieldSign.SIGNED : (byte)BitFieldSign.UNSIGNED;
```

Comment:
can we use `encoding[0] is 'i' or 'I' ? (byte)BitFieldSign.SIGNED : (byte)BitFieldSign.UNSIGNED;` here?

Reply:
cc @vazois